connectivity/FEATURE_BLE/libraries/cordio_stack/ble-host/sources/stack/dm/dm_sec.c

```diff
@@ -368,17 +368,17 @@ void DmSecSetLocalIrk(uint8_t *pIrk)
 /*!
  *  \brief  This function sets the local identity address used by the device.
  *
- *  \param  pAddr     Pointer to the address.
- *  \param  type      Type of the address.
+ *  \param  identityAddr  Local identity address.
+ *  \param  addrType      Local identity address type.
  *
  *  \return None.
  */
 /*************************************************************************************************/
-void DmSecSetLocalIdentityAddr(const uint8_t *pAddr, uint8_t type)
+void DmSecSetLocalIdentityAddr(const bdAddr_t identityAddr, uint8_t addrType)
```

Comment:
It's interresting that bdAddr is different from identity address now. This may have an impact with code in BLE API.